### PR TITLE
Merge deputy and lead roles

### LINKED ISSUE #364: Update roles list to combine Lead/Deputy
As discussed and unanimously agreed to at the Astropy Coordination Meeting, we are combining the "lead" and "deputy" columns in the roles list.

This is conceptually straightforward but requires some re-work of the javascript...

### PR DESCRIPTION
**Pull request by astrofrog**
As decided at the Coordination meeting in Socorro. I'll also email astropy-core-maintainers to let people know about the change (done!)

Fixes #364

### CODE REVIEW

File: roles.json [hunk 1]
```diff
@@ -299,103 +278,83 @@
...
                 "role": "astropy.convolution",
-                "lead": ["Adam Ginsburg"],
-                "deputy": ["Axel Donath", "Larry Bradley"]
+                "people": ["Adam Ginsburg", "Axel Donath", "Larry Bradley"]
             },
             {
                 "role": "astropy.coordinates",
-                "lead": ["Erik Tollerud"],
-                "deputy": ["Stuart Littlefair", "Adrian Price-Whelan", "Juan Luis Cano Rodr\u00edguez"]
+                "people": ["Erik Tollerud", "Stuart Littlefair", "Adrian Price-Whelan", "Juan Luis Cano Rodr\u00edguez"]
```
Comment (mhvk) [inline roles.json:289]: Might as well add me here too...

**Comment by mhvk** [comment]
Second small comment: Add "at least" before the "1" in `Sub-package maintainer (1 per core package sub-package)`.

File: roles.json (hunk 1 above)
Comment (astrofrog) [reply roles.json:289]: Done!

**Comment by astrofrog** [comment]
@mhvk - done!

File: roles.json [hunk 2]
```diff
@@ -295,107 +274,87 @@
...
                 "role": "astropy.io.misc",
-                "lead": ["Unfilled"],
-                "deputy": ["Matteo Bachetti", "Ed Slavich"]
+                "people": ["Matteo Bachetti", "Ed Slavich"]
             },
             {
                 "role": "astropy.io.votable",
-                "lead": ["Pey Lian Lim"],
-                "deputy": ["Thomas Boch", "Tom Donaldson", "Pey Lian Lim"]
+                "people": ["Pey Lian Lim", "Thomas Boch", "Tom Donaldson", "Pey Lian Lim"]
```
Comment (pllim) [inline roles.json:309]: No need to list me twice. 😅 
```suggestion
                "people": ["Thomas Boch", "Tom Donaldson", "Pey Lian Lim"]
```

**Comment by astrofrog** [comment]
@pllim - I've now sorted alphabetically by last name (@mhvk - should we put you at K or V?)

>    sub-package -> subpackage

I'm not sure I agree, let's leave to another PR since unrelated

>    I thought @Cadair is the Matrix coordinator? Do we even need a Slack coordinator?

Yes added. We do need someone to be in charge of slack and to moderate. Unless you are suggesting we drop Slack and switch to Matrix? :trollface: 

>    There should be some order to the names; maybe alphabetical by last names?

Done, by last name

**Comment by pllim** [comment]
Re: sub-package vs subpackage -- See https://docs.astropy.org/en/latest/development/style-guide.html

Re: Slack -- Currently crowd sourcing seems okay...? Impossible for one person to moderate all the channels.

**Comment by mhvk** [comment]
The ordering makes me wonder a bit about whether it is in fact a good idea to change the scheme, since for some subpackages I do less than for others: for all I feel comfortable doing all the review, etc., but for `units` I feel a much stronger obligation to review and more of a sense that I should be involved in larger decisions.

I also realized I have quoted my role on my CV, as a way to show my efforts are recognized, and it definitely makes it easier with PI/Co-I type of divisions.

That said, I wasn't there at the coordination meeting, so this may simply come too late. If so, alphabetize me under "v" (it is under "k" in NL, but always under "v" in english-speaking countries, so that makes more sense).

**Comment by pllim** [comment]
Re: https://github.com/astropy/astropy.github.com/pull/380#issuecomment-629461125 -- Is there a way to highlight domain experts or is that a step backward? 💭

**Comment by astrofrog** [comment]
@mhvk: to some extent the requirements for review and so on are going to be very sub-package dependent - for example for astropy.time you might want your approval to be required for any PR, whereas for astropy.coordinates that might not be the case. This problem wasn't really solved by the lead/deputy structure, and instead I think it's up to sub-package maintainers to decide how they want to operate. However, I understand the point about the CV/credit, but would this be alleviated somewhat if we titled this role 'sub-package lead maintainers'? (i.e. adding 'lead')

**Comment by mhvk** [comment]
@astrofrog - fair enough. If we're not making a distinction, then I would leave the titles as they are. 

On second thought, I think it is indeed clear enough - all the astropy maintainers can in principle review PRs for any module, but, as you note, for anything tricky they might not want to just approve.

**Comment by mhvk** [comment]
p.s. And `coordinates` is just a very big beast, so having more there is not strange...

**Comment by astrofrog** [comment]
No objections to this from anyone in roles based on my email to the group, and @kelle and @taldcroft approved this verbally at the last CoCo meeting ,so I'll go ahead and merge tomorrow unless someone beats me to it since this had some conflicts and was a pain to rebase 😄

**Comment by taldcroft** [comment]
@astrofrog - I went ahead and merged since everyone was on-board with the merge.

**Pull request merged**